### PR TITLE
CAL-1414 All day is returned to unchecked after check in schedule popup

### PR DESCRIPTION
**Pull request by nttuyen**
- Re-calculate checkbox Allday when the date are updated

### CODE REVIEW

File: calendar-webapp/src/main/webapp/vue-app/components/ExoCalendarEventForm.vue [hunk 1]
```diff
@@ -382,6 +382,7 @@ export default {
     saveFindTime(evt) {
       this.event.fromDate = new Date(evt.from.getTime());
       this.event.toDate = new Date(evt.to.getTime());
+      this.isAllDay = this.event.isAllDay();
```
Comment (thomasdelhomenie) [inline calendar-webapp/src/main/webapp/vue-app/components/ExoCalendarEventForm.vue]: Shoudn't we bind allDay like we do for fromDate and toDate ?

Comment (nttuyen) [reply calendar-webapp/src/main/webapp/vue-app/components/ExoCalendarEventForm.vue]: - isAllDay is model of the checkbox, when this checkbox is checked, the fromDate and toDate will be updated to show a full-day, and also disable those date inputs
- When this checked is un-checked, date fields are enable
--> mean that this field model is currently used for handling click action of user. If we bind this field (this field will be computed field) , we will have to bind click action to a handler and handle it base on the checkbox checked status
--> is that ok ? pls confirm , thank you

Comment (thomasdelhomenie) [reply calendar-webapp/src/main/webapp/vue-app/components/ExoCalendarEventForm.vue]: Cannot we bind the checkbox on event.isAllDay and update the :disabled attribute of dates comboboxes to event.isAllDay ?

Comment (nttuyen) [reply calendar-webapp/src/main/webapp/vue-app/components/ExoCalendarEventForm.vue]: @thomasdelhomenie , as discussed, i've add comments to make it clearer, pls check

**Pull request merged**